cmd/nvkind/cluster_create.go

```diff
@@ -19,14 +19,25 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"io"
+	"log"
```

Comment:
I would use klog/v2 -- mostly because it is already in use in main.go and is much more flexible if we want to use its more advanced features in the future.

Reply:
done